cli/src/commands/git/fetch.rs

```diff
@@ -251,3 +250,20 @@ fn warn_if_branches_not_found(
 
     Ok(())
 }
+
+fn warn_ignored_refspecs(
+    ui: &Ui,
+    remote_name: &RemoteName,
+    IgnoredRefspecs(ignored_refspecs): IgnoredRefspecs,
+) -> Result<(), CommandError> {
+    let remote_name = remote_name.as_symbol();
+    for IgnoredRefspec { refspec, reason } in ignored_refspecs {
+        writeln!(
+            ui.warning_default(),
+            "ignored refspec \"{}\" from `{remote_name}`: {reason}",
```

Comment:
"Ignored refspec `{refspec}` from ..."

* start with capital letter, which is our convention
* use backtick consistently
* `BString` implements `Display`

Reply:
also use a raw string here to avoid \-escaping the quotation marks

Reply:
> * `BString` implements `Display`

Ah I'll remember this one thank you! This is fixed